markdownlint/TOP010_useLazyNumbering/TOP010_useLazyNumbering.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  names: ["TOP010", "ordered-list-for-lazy-numbering"],
+  description: "Ordered lists should use lazy numbering",
+  information: new URL(
+    "https://github.com/TheOdinProject/curriculum/blob/main/markdownlint/docs/TOP010.md"
+  ),
+  tags: ["lazy-numbering"],
```

Comment:
[Markdownlint rules tags](https://github.com/DavidAnson/markdownlint/tree/v0.32.1?tab=readme-ov-file#tags).
Not that big of a deal considering we're not really going to be using the tags much, but for consistency, it'll be good to use the existing rule tags.
```suggestion
  tags: ["ol"],
```